templates/rhdm71-prod-kieserver.yaml

```diff
@@ -0,0 +1,669 @@
+      to manage this Decision Server, set this variable to the OpenShift service name for the Decision Central. In this
+      case, do not set the KIE server controller protocol, KIE server controller host, and KIE server controller port.
+      (Sets the org.kie.server.controller system property if host and port are not set).
+  name: KIE_SERVER_CONTROLLER_PROTOCOL
+  value: http
+  required: false
+- displayName: KIE server controller service
+  description: The service name for the optional Decision Central or controller, where it can be reached and registered with. (Used to set the org.kie.server.controller system property if host and port aren't set)
+  name: KIE_SERVER_CONTROLLER_SERVICE
+  value: ''
+  required: false
+- displayName: KIE server controller host
+  description: KIE server controller host. To use Decision Central or a controller to manage this Decision
+    Server, set this parameter to the host part of the URL for the Decision Central or controller. (Used to 
+    set the org.kie.server.controller system property).
+  name: KIE_SERVER_CONTROLLER_HOST
+  example: my-app-controller-ocpuser.os.example.com
+  required: false
+- displayName: KIE server controller port
+  description: KIE server controller port. To use Decision Central or a controller to manage this Decision
+    Server, set this parameter to the port part of the URL for the Decision Central or controller. (Used to 
+    set the org.kie.server.controller system property).
+  name: KIE_SERVER_CONTROLLER_PORT
+  example: '8080'
+  required: false
+- displayName: Drools Server Filter Classes
+  description: KIE server class filtering (Sets the org.drools.server.filter.classes system property)
+  name: DROOLS_SERVER_FILTER_CLASSES
+  value: 'true'
+  required: false
+- displayName: KIE MBeans
+  description: KIE server mbeans enabled/disabled (Sets the kie.mbeans and kie.scanner.mbeans system properties)
+  name: KIE_MBEANS
+  value: enabled
+  required: false
+- displayName: KIE server http Route Hostname
+  description: 'Custom hostname for http service route, if set will also configure the KIE_SERVER_HOST. Leave blank for default hostname,
+    e.g.: <application-name>-kieserver-<project>.<default-domain-suffix>'
+  name: KIE_SERVER_HOSTNAME_HTTP
+  value: ''
+  required: false
+- displayName: KIE server Custom https Route Hostname
+  description: 'Custom hostname for https service route.  Leave blank for default
+    hostname, e.g.: secure-<application-name>-kieserver-<project>.<default-domain-suffix>'
+  name: KIE_SERVER_HOSTNAME_HTTPS
+  value: ''
+  required: false
+- displayName: KIE server 2 Custom http Route Hostname
+  description: 'Custom hostname for http service route, if set will also configure the KIE_SERVER_HOST. Leave blank for default hostname,
+    e.g.: <application-name>-kieserver-<project>.<default-domain-suffix>'
+  name: KIE_SERVER2_HOSTNAME_HTTP
```

Comment:
If I am right, in this template is deployed only one Kie Server. So please delete properties for KIE_SERVER2

Reply:
That's right, @errantepiphany  shouldn't this template include 2 kie-servers?